test/cli_dfget_p2p_test.go

```diff
@@ -114,3 +114,65 @@ func (s *DFGetP2PTestSuite) TestDownloadFile(c *check.C) {
 		}
 	}
 }
+
```

Comment:
The integration test here is annotated due to the reason that the detailed implementation of IPC between user and DFGET demands further discussion.